test/studies/isx/.gitignore

```diff
@@ -0,0 +1 @@
+*.good
```

Comment:
(For the github record: On chat, a few opinions in favor of decentralized .gitignores came in, none opposed)